bin/odyssey/src/main.rs

```diff
@@ -124,6 +125,12 @@ fn main() {
                 })
                 .await?;
 
+            // spawn raw transaction forwarding
+            let txhandle = node.node.network.transactions_handle().await.unwrap();
+            let raw_txs =
+                node.node.add_ons_handle.eth_api().eth_api().subscribe_to_raw_transactions();
```

Comment:
`eth_api().eth_api()` we should fix

Reply:
not unless we want to replace the wrapped `OpEthApi(EthApi)` with actually impls for OpEthApi, but I guess we could rename this fn instead